oot.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/bash
+
+
+function usage {
+	echo 'Usage:'
+	echo -e "\t$0 <device>\t\t(Example for akatsuki: $0 akatsuki)"
+	echo -e "\t$0 -c gcc <device>\t(Will build with GCC toolchain)"
+	echo -e "\t$0 -f <device>\t\t(Will flash the kernel using fastboot)"
```

Comment:
I much prefer writing out every individual flag, and optionally have some examples at the bottom. Something like:

```
USAGE:
	$0 [FLAGS] <device> [<device>...]

	Perhaps explain what to pass in as device(s).

FLAGS:
	-c		Select a compiler to use. Currently supported compilers are gcc, linaro_gcc and clang (defaults to clang).
	-f		Flash the kernel after compiling

EXAMPLES:
	$0 akatsuki
	etc
```

I'm still disappointed optargs can't generate such a thing for us... I've gotten a bit too used to full-fledged argument parsers with typing, validation, help and everything baked right in (Python `argparse`, Rust `clap`).

Reply:
I just copied your format :)

Reply:
Fine!

Unfortunately the size of `$0` screws up tab indentation... `./oot.sh` is fine, but `./oot/oot.sh` makes the examples misaligned :/